src/PostgREST/Types.hs

```diff
@@ -163,7 +163,13 @@ operators = M.fromList [
   ("is", "IS"),
   ("@@", "@@"),
   ("@>", "@>"),
-  ("<@", "<@")]
+  ("<@", "<@"),
+  ("&&", "&&"),
```

Comment:
That works but it does introduce a breaking change since it would replace `@@`, which has been implemented for a while now.

My preference would be to also keep using symbols as much as possible for these.